runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -5611,20 +5613,68 @@ chpl_comm_nb_handle_t chpl_comm_put_nb(void* addr, c_nodeid_t node,
   if (put_prologue(addr, node, raddr, size, commID, ln, fn)) {
     handle = ofi_put_nb(handle, addr, node, raddr, size);
   }
+  chpl_comm_diags_incr(put_nb);
```

Comment:
this will now count all puts as a comm diag right, even if its a local put thats done as a memcpy?

Reply:
Good catch. I've fixed the counters for both PUTs and GETs.